### PR TITLE
Add GraphQL code generation to actions-server

### PR DESCRIPTION
**Pull request by namoscato**
* Move most of the GraphQL Code Generator configuration to the root directory
* Add necessary dev dependencies to `actions-server`

### CODE REVIEW

File: actions-server/package.json [hunk 1]
```diff
@@ -18,10 +19,17 @@
...
+    "@graphql-codegen/cli": "^1.14.0",
+    "@graphql-codegen/typescript": "^1.14.0",
+    "@graphql-codegen/typescript-operations": "^1.14.0",
+    "@graphql-codegen/typescript-react-apollo": "^1.14.0",
     "@types/express": "^4.17.6",
     "@types/jsonwebtoken": "^8.5.0",
     "@types/node": "^14.0.1",
     "@types/node-fetch": "^2.5.7",
+    "@types/react": "^16.9.35",
+    "apollo-client": "^2.6.10",
```
Comment (namoscato) [inline actions-server/package.json:32]: It's a little weird the server depends on some of these things but 🤷‍♂️

File: app/src/generated/graphql.schema.json [hunk 2]
```diff
@@ -5200,6 +5200,16 @@
...
+          {
+            "name": "created_at",
+            "description": "",
+            "type": {
+              "kind": "INPUT_OBJECT",
+              "name": "timestamptz_comparison_exp",
+              "ofType": null
+            },
+            "defaultValue": null
+          },
```
Comment (namoscato) [inline app/src/generated/graphql.schema.json:5212]: I didn't look into why this changed...

File: actions-server/package.json (hunk 1 above)
Comment (avimoondra) [reply actions-server/package.json:32]: i think a couple options here.. have two different codegen, which generate only the necessary code in each repo, or a new top level directory that just does codegen, and app/ & actions-server/ reference that.

**Comment by avimoondra** [comment]
closing in favor of https://github.com/avimoondra/fishbowl/pull/74, and a poor git form on my part... 🤦

**Pull request closed without merging**